lib/loadGltfUris.js

```diff
@@ -41,23 +42,26 @@ function loadURI(gltf, basePath, name, callback) {
             var object = objects[id];
             var uri = object.uri;
             object.extras = defaultValue(object.extras, {});
+            object.extras._pipeline = defaultValue(object.extras._pipeline, {});
```

Comment:
No need to change this now, but just a heads up that this just adds `extras._pipeline` to buffers, images, and shaders.  In the future, I suspect we will want a stage early in the pipeline to add it to all objects so later stages know it will be there, and then a stage at the end of the pipeline to remove it (and `extras` when possible) from all objects.